Controller/NewsController.php

```diff
@@ -71,8 +77,15 @@ public function getNewsListAction(Request $request)
         $restHelper = $this->get('sulu_core.doctrine_rest_helper');
         $factory = $this->get('sulu_core.doctrine_list_builder_factory');
 
+        $fieldDescriptors = $this->getFieldDescriptors();
         $listBuilder = $factory->create(self::ENTITY_NAME);
-        $restHelper->initializeListBuilder($listBuilder, $this->getFieldDescriptors());
+        $restHelper->initializeListBuilder($listBuilder,$fieldDescriptors);
+
+        if (null === $request->get('sortBy')) {
+            // default sort-by releaseDate ASC
+            $listBuilder->sort($fieldDescriptors['releaseDate']);
```

Comment:
if no `sortBy` isset on the request - the default sorting should be by `releaseDate`